src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double height;
+
+    public Rectangle(String color, double length, double height) {
+        super(color);
+        this.length = length;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return length * height;
+    }
+
+    public void draw() {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. Since `draw()` and `calculateArea()` are conceptually different, consider separating them into different interfaces.